app/backend/requirements.txt

```diff
@@ -1,9 +1,5 @@
-#
-# This file is autogenerated by pip-compile with Python 3.12
-# by the following command:
-#
-#    pip-compile requirements.in
-#
+# This file was autogenerated by uv via the following command:
+#    uv pip compile requirements.in -o requirements.txt
```

Comment:
Pip was unable to resolve the dependencies, uv had no problems and was super fast. So we can use uv to compile, and you can still use pip to install.